clawhub/honcho-setup/SKILL.md

```
@@ -1,50 +1,62 @@
+      data_destination_purpose: "Migrates file-based memory system to Honcho API for AI agent memory/personalization"
   homepage: "https://honcho.dev"
   source: "https://github.com/plastic-labs/honcho"
 ---
 
 # Honcho Setup
 
-Install the Honcho plugin and migrate legacy workspace memory files to Honcho.
+Install the `@honcho-ai/openclaw-honcho` plugin. The plugin includes `openclaw honcho setup`, which handles API key configuration and migration of any legacy memory files to Honcho.
+
+> ⚠️ **DATA UPLOAD WARNING**: `openclaw honcho setup` (run in Step 2) will offer to upload your workspace memory files (USER.md, MEMORY.md, IDENTITY.md, memory/, canvas/, SOUL.md, AGENTS.md, BOOTSTRAP.md, TOOLS.md) to an external API. `HEARTBEAT.md` is excluded. By default, data is sent to `api.honcho.dev`. For self-hosted instances, data is sent to your configured `HONCHO_BASE_URL`. The setup command will show exactly which files will be uploaded and ask for explicit confirmation before proceeding.
 
```

markdownlint-cli2 (0.21.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)